src/test/java/teammates/test/driver/GaeSimulation.java

```diff
@@ -240,12 +242,34 @@ private HttpServletRequest createWebRequest(String uri, String... parameters) {
     public static Map<String, Object> getEnvironmentAttributesWithApplicationHostname() {
         Map<String, Object> attributes = new HashMap<>();
         try {
+            if (!TestProperties.TEAMMATES_URL.contains("localhost")) {
```

Comment:
Instead of changing this. I think we can change the `AdminHomePageUiTest` to only comparator param ignore hostname.